upath/registry.py

```diff
@@ -64,6 +64,7 @@ class _Registry(MutableMapping[str, "type[upath.core.UPath]"]):
         "adl": "upath.implementations.cloud.AzurePath",
         "az": "upath.implementations.cloud.AzurePath",
         "file": "upath.implementations.local.LocalPath",
+        "local": "upath.implementations.local.LocalPath",
```

Comment:
In `upath/tests/test_registry.py` "local" has to be added to `IMPLEMENTATIONS` too.